src/znc.cpp

```diff
@@ -1955,10 +1935,8 @@ void CZNC::ResumeConnectQueue() {
 
 void CZNC::AddNetworkToQueue(CIRCNetwork *pNetwork) {
 	// Make sure we are not already in the queue
-	for (list<CIRCNetwork*>::const_iterator it = m_lpConnectQueue.begin(); it != m_lpConnectQueue.end(); ++it) {
-		if (*it == pNetwork) {
-			return;
-		}
+	if (std::find(m_lpConnectQueue.begin(), m_lpConnectQueue.end(), pNetwork) != m_lpConnectQueue.end()) {
```

Comment:
More modern way of writing this would be `std::begin(m_lpConnectQueue)` and `std::end(m_lpConnectQueue)`. I don't know what would be the benefit though.